### PR TITLE
fix: default list arg

### PR DESCRIPTION
**Pull request by unparalleled-js**
fixes: #15

### CODE REVIEW

**Comment by unparalleled-js** [comment]
I am converting to draft because I am not sure if it needs to be like this for RLP encoding reasons.

**Comment by mikeshultz** [comment]
I'm not sure this really has any impact here.  If you provide a non-empty list it should error since that type hasn't been fleshed out yet (will be part of the type 1 transaction work).  And when it is implemented, I doubt it modifies the list at all.

That said, I'd be okay to merge this just for good code hygiene.  But that type hint should probably be `Optional[List[bytes]]` in that case.

File: ledgereth/transactions.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def create_transaction(
     max_fee_per_gas: int = 0,
     chain_id: int = DEFAULT_CHAIN_ID,
     sender_path: str = DEFAULT_PATH_STRING,
-    access_list: Optional[List[Tuple[bytes, List[int]]]] = None,
+    access_list: Optional[List[Tuple[Union[bytes, str], List[int]]]] = None,
```
[the remark this answers is not in the document]
Comment (mikeshultz) [reply ledgereth/transactions.py:153]: Yeah, you're correct.  `str` is acceptable here and types are coerced as needed.

**Comment by mikeshultz** [comment]
Sorry, I missed your updates to this PR.  I do think this fix is useful and fixes a potentially painful edge case.  There's been a bunch of updates since you first created this including new Serializable objects with this same issue.  Let me know if you'd like to finish this up.

**Comment by mikeshultz** [comment]
Looks like there's a couple conflicts and some lint (`python setup.py lint` would take care of it).  If you take care of those I'll get this merged in.

**Comment by mikeshultz** [comment]
Thanks for raising and solving this issue!  :+1:

**Pull request merged**